presto-tests/src/test/java/com/facebook/presto/tests/TestTpchDistributedStats.java

```diff
@@ -44,20 +46,29 @@
 import static java.util.stream.Collectors.groupingBy;
 import static org.testng.Assert.assertEquals;
 
+@Test(singleThreaded = true)
 public class TestTpchDistributedStats
 {
     public static final int NUMBER_OF_TPCH_QUERIES = 22;
 
-    DistributedQueryRunner runner;
+    private DistributedQueryRunner runner;
 
-    public TestTpchDistributedStats()
+    @BeforeClass
+    public void init()
             throws Exception
     {
         runner = createQueryRunnerWithoutCatalogs(emptyMap(), emptyMap());
         runner.createCatalog("tpch", "tpch", ImmutableMap.of(
                 "tpch.column-naming", ColumnNaming.STANDARD.name()));
     }
 
+    @AfterClass(alwaysRun = true)
+    public void shutdown()
```

Comment:
nit: these are usually called `setUp` and `tearDown` in tests, i believe

Reply:
I see a mix of those, but I guess setup/teardown are more common. thanks!